### PR TITLE
Add links to votable arbitrator election ballot(s)

### LINKED ISSUE #310: Add ballot links for Resolve arbitrator elections
## Overview
https://app-dev.telos.net/resolve/elections
Currently it is unclear how to navigate to a ballot to either a.) vote or b.) review the results
image

## Acceptance Criteria
A clickable links that takes the user to ___/trails/ballot/{{ballot_name}}

### PR DESCRIPTION
**Pull request by kylanhurt**
# Fixes #310 

## Description

Simply added a clickable link to `/trails/ballot/{ballot_name}`

## Test scenarios

Connected to testnet where there are elections and made sure the links worked

## Checklist:
-   [x] I have performed a self-review of my own code
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file and/or created a new issue with the required translations for the currently supported languages
-   [x] I have tested for mobile functionality and responsiveness

### CODE REVIEW

File: src/pages/resolve/components/ElectionsTable.vue [hunk 1]
```diff
@@ -29,10 +29,18 @@
                         <q-td key="ballot_name" class="ballot_name">
                             <div class="info">
                                 <div class="ballot-name">
+                                  <a v-if="props.row.ballot_name" :href="`/trails/ballot/${props.row.ballot_name}`">
                                     {{
                                         props.row.ballot_name ||
```
Comment (donnyquixotic) [inline src/pages/resolve/components/ElectionsTable.vue:34]:
```suggestion
                                        props.row.ballot_name 
```
a tag is only displayed if `props.row.ballot_name` exists so no need for condition

File: src/pages/resolve/components/ElectionsTable.vue [hunk 2]
```diff
@@ -29,10 +29,18 @@
...
                                             `${props.row.election_id}`
```
Comment (donnyquixotic) [inline src/pages/resolve/components/ElectionsTable.vue:35]:
```suggestion
```
see previous suggestion, will never reach condition

File: src/pages/resolve/components/ElectionsTable.vue [hunk 3]
```diff
@@ -29,10 +29,18 @@
...
                                     }}
+                                  </a>
+                                  <span v-else>
+                                    {{
+                                        props.row.ballot_name ||
```
Comment (donnyquixotic) [inline src/pages/resolve/components/ElectionsTable.vue:40]:
```suggestion
```
unnec condition, will display link if exists

**Review by donnyquixotic — APPROVED** [review]

**Comment by kylanhurt** [comment]
Closing because arbitration is being put on hold for now

**Pull request closed without merging**